tsl/test/sql/CMakeLists.txt

```diff
@@ -104,7 +101,11 @@ endif(CMAKE_BUILD_TYPE MATCHES Debug)
 if((${PG_VERSION_MAJOR} GREATER_EQUAL "14"))
   if(CMAKE_BUILD_TYPE MATCHES Debug)
     list(APPEND TEST_FILES chunk_utils_internal.sql)
+    list(APPEND TEST_TEMPLATES continuous_aggs.sql.in
+         continuous_aggs_deprecated.sql)
   endif()
+  list(APPEND TEST_FILES compression.sql compression_upd_del.sql
```

Comment:
shouldn't these tests be executed also for pg versions < 14?

Reply:
We don't support UPDATE/DELETE operation on PG version < 14, this is because planner does not generate CustomScan (HypertableModify) node on these versions.

These tests have several queries related to UPDATE/DELETE operations on compressed hypertables, if we want to run them we need different result (.out) file for version < 14
Instead i decided to run them only on version >= 14